pkg/apis/kops/cluster.go

```diff
@@ -207,6 +207,22 @@ type ClusterSpec struct {
 	ClusterAutoscaler *ClusterAutoscalerConfig `json:"clusterAutoscaler,omitempty"`
 	// WarmPool defines the default warm pool settings for instance groups (AWS only).
 	WarmPool *WarmPoolSpec `json:"warmPool,omitempty"`
+
+	// IAMRolesForServiceAccount defines the IRSA configuration.
```

Comment:
```suggestion
	// IAMRolesForServiceAccount defines external permission roles for service accounts.
```

The consensus was to name this to indicate it is provisioning external roles. I question the need to create a new substruct containing a single field. I think we should either create the array at top level or put it in `iam` with a clearer name. So something like `serviceAccountIAMRoles` or `iam.serviceAccountRoles`.

Reply:
Looks like I had a bad merge here. They should have been prefixed.